data/autoyast_sle12/autoyast_sle12_eula.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
```

Comment:
Would you mind renaming that autoyast profile to a name without the version included? It looks generic and version agnostic which is good so we can hope it also works for sle13 without changes. We want to avoid calling something that is called "sle12" for "sle13" though to avoid confusion :-)